### DISCUSSION TITLE
Seurat Subset Removal Error

### DISCUSSION THREAD

**Discussion by angiemcgraw** [satijalab/seurat#6503] [Q&A]
Hi,
I'm trying to subset my Seurat object (cells that are not part of cluster 14, in this case):
`subsetted_seurat_obj <- subset(seurat_obj, subset = seurat_clusters != 14)`

seurat_clusters is under the seurat_obj's meta.data: 
`seurat_obj@meta.data$seurat_clusters`

But, I received the following error: 
`Error in intI(i, n = d[1L], dn[[1L]], give.dn = FALSE) :  invalid character indexing`

I'm not sure how to fix this, any help would be appreciated. Thank you!

Angie

**Comment by adRn-s** [comment]
related/ answered: https://github.com/satijalab/seurat/issues/3030#issuecomment-893484872
